packages/mobile/src/utils/formatting.ts

```diff
@@ -10,7 +10,8 @@ export const getMoneyDisplayValue = (
   includeSymbol: boolean = false,
   roundingTolerance: number = 1
 ): string => {
-  const decimals = CURRENCIES[currency].displayDecimals
+  // TODO: remove the hardcoded ternary operation here when displayDecimals is updated from monorepo
+  const decimals = currency === CURRENCY_ENUM.GOLD ? 4 : CURRENCIES[currency].displayDecimals
```

Comment:
should it be 4 decimal places for CELO transactions? https://github.com/celo-org/celo-monorepo/issues/4713 says 3

Reply:
Had a discussion with Aaron and Corey about this and ultimately decided on 2 for fiat and 4 for CELO -- I think Aaron will know more about the rationale behind that!